parser.js

```diff
@@ -58,6 +58,7 @@
 //      DecimalEscape
 //      CharacterEscape
 //      CharacterClassEscape
+//      k GroupName
```

Comment:
For understanding: the `k` is matching the character `k` here?

Reply:
Yes :+1: